website/app/styles/pages/application/sidebar.scss

```diff
@@ -1,21 +1,22 @@
 // SIDE NAVIGATION
 
-$docs-page-sidebar-width: 241px;
-
 .doc-page-sidebar {
-  flex: none;
-  // position: fixed;
-  // top: 0;
-  // bottom: 0;
-  // left: 0;
-  width: $docs-page-sidebar-width;
-  padding: 2rem 0.5rem;
-  background: white;
-  border-right: 1px solid #ddd;
+  display: none; // TODO! - We have to implement the whole "mobile" experience, for now we simply hide it
+
+  @media (min-width: 1000px) {
+    display: flex;
+    flex-direction: column;
+    grid-area: sidebar;
+    width: var(--doc-page-sidebar-width); // TODO discuss with Heather if we want to reduce this width when the viewport is "medium"
+    background: white;
+    border-right: 1px solid #ddd;
+  }
 }
 
-.doc-page-sidebar__table-of-contents {
-  // width: 100%;
-  // overflow: hidden;
-  // overflow-y: auto;
+.doc-page-sidebar__inner-wrapper {
+  position: relative;
```

Comment:
note: I don't follow why we have this down as relative, but if we need it so and `.doc-page-header` ends up sticky (as it currently is) we'll have to set a `z-index` or so on `.doc-page-header` so it doesn't overlap with the header when the whole page scrolls

Reply:
Agree. Right now this `position: relative;` doesn't seem needed and it's causing a z-index issue if you scroll the sidebar text.

Reply:
Yes, you're right @alex-ju @KristinLBradley, didn't look much in the the navigation because will be finalized later (designs are not final), but I should have catched that one. Sorry :) 

Fixed in https://github.com/hashicorp/design-system/pull/697/commits/4487b714a3b701bb9f1fbe41d2196e353098c5a1

PS: @KristinLBradley I assume you approve the PR since you didn't add more comments (in that case if in the future you can explicitly approve the PRs would be better, so I know if I can merge or not; for now I'll merge so this unblocks other work; in case you have comments you can add them here and I'll address in a follow-up PR).

Reply:
Sorry I was waiting for your response to this one. I'm not used to using github for PR reviews so maybe I'm not submitting reviews properly. (I'm used to adding comments if I have any, waiting to see the person's response and then if I'm ok with it approving it.)